### PR TITLE
layerscape: add 5.15 kernel

### PR DESCRIPTION
**Pull request by CHKDSK88**
Run tested: LS1012A-FRDM, LS1046A-RDB (nor and sd-boot images), LS1021A-IOT (By @ArtelMike with u-boot fix #11102)

~~Known problems (on 5.10 are the same issues):~~
-~~SPI NOR not working in LS1012A and  LS1046A  (uncorrect data read and uncorrect write on LS1012A-FRDM and LS1046A-RDB)~~
-~~SD card make boot impossible (LS1046A-RDB)~~ (My SD card slot was broken)
~~It will need some time to resolve it.~~

### CODE REVIEW

**Comment by CHKDSK88** [comment]
@yangbolu1991 @sch-m 
Could You test and help resolve issues?

**Comment by CHKDSK88** [comment]
@ArtelMike Could You test 5.15 on LS1021A-IOT (with u-boot fix)?

**Comment by ArtelMike** [comment]
@CHKDSK88 Please point me to the commit to use for 5.15?

Mike

**Comment by CHKDSK88** [comment]
Thanks @ArtelMike. It's ready for wide testing now.

File: target/linux/layerscape/patches-5.10/305-arm64-dts-ls1046a-rdb-Update-qspi-spi-rx-bus-width-t.patch [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ 1 file changed, 2 insertions(+), 2 deletions(-)
+
+--- a/arch/arm64/boot/dts/freescale/fsl-ls1046a-rdb.dts
++++ b/arch/arm64/boot/dts/freescale/fsl-ls1046a-rdb.dts
+@@ -101,7 +101,7 @@
+ 		#address-cells = <1>;
+ 		#size-cells = <1>;
+ 		spi-max-frequency = <50000000>;
+-		spi-rx-bus-width = <4>;
++		spi-rx-bus-width = <1>;
```
Comment (hauke) [inline target/linux/layerscape/patches-5.10/305-arm64-dts-ls1046a-rdb-Update-qspi-spi-rx-bus-width-t.patch:22]: It looks like quad SPI is not deactivated in two places.
It is deactivated for some boards by cahnging to `spi-rx-bus-width = <1>` and with the patch for the `spansion.c` file. 
Is this all needed?

Comment (CHKDSK88) [reply target/linux/layerscape/patches-5.10/305-arm64-dts-ls1046a-rdb-Update-qspi-spi-rx-bus-width-t.patch:22]: @hauke 
For unknown for me reason, it require both patches.

**Comment by hauke** [comment]
Thank you for the patches, I applied them to master in 2e4fe289ceb02139e6f1cf8dcae31ad14efba52c, 2268d555218e3228705beda65e6557ddfc5cf18d, f620fb4b7389e1d02791f51e38c6e57920ad39dc, 8efffce6779856874b1e33a4d50ddce5f9c2548f, a42edce9fdeeecd87dc62fb5573ac6c0e610c000, d64d1c86cce77b052be97f281d3a424c89f292df, 23b9b01cdd612bfa46620cd505285707b164e891 and 5b0e111789314c9ff0f351e3b5d54d43d4485813

**Pull request closed without merging**